templates/configmap.yaml

```diff
@@ -12,7 +12,7 @@ data:
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: storage
+  name: {{ .Release.Name }}-storage
```

Comment:
I think since this uses `storage` elsewhere this might be better named `-database` now? To avoid confusion?

Reply:
Since the previous name was `storage` and the `ConfigMap` contains:
- `PHOTOPRISM_DATABASE_DRIVER`
- `PHOTOPRISM_ORIGINALS_PATH`
- `PHOTOPRISM_IMPORT_PATH`
- `PHOTOPRISM_STORAGE_PATH` 

which are all storage related, I think `storage` is a good suffix. But I'm happy to change it to `database` if you have strong opinions about it 🙂

Reply:
Yeah I think you're right. 👍